### PR TITLE
Add support to rails 5

### PR DESCRIPTION
**Pull request by chucre**
This method is not more available.

### CODE REVIEW

**Comment by viacommits** [comment]
Hey @chucre, if you're interested, I have an alternate gem for Rails 5+ built from declarative authorization, https://github.com/tektite-software/authoreyes

It should be more or less a drop-in replacement.

**Comment by OlivierGrimard** [comment]
`+ 1`

**Comment by ipmsteven** [comment]
@zeiv Does that mean all users of declarative-authorization should migrate to authoreyes if they want use it with rails 5? what's the plan for declarative-authorization?

**Comment by viacommits** [comment]
@ipmsteven I don't know.  I was just about to submit a PR updating the Readme to officially deprecate Declarative Authorization in favor of Authoreyes.  I was waiting to submit this until I have fleshed out the Readme for Authoreyes, which is basically fully functional.

I am not _officially_ affiliated with Declarative Authorization or one of it's maintainers, but IMHO if you like Declarative Authorization and are using Rails 5, Authoreyes is the way to go.

Authoreyes isn't Declarative Authorization updated for Rails 5.  It is a new authorization solution with two primary goals: 1) Secure, modern authorization and 2) Drop-in replacement for Declarative Authorization.

Basically, if you have a Rails app using DA and are updating to Rails 5, Authoreyes will be the most pain-free solution.  I haven't done a ton of research into this, but as far as I can tell Authoreyes is the only authorization gem for Rails that uses the same approach as Declarative Authorization.

I hope that helps.

**Comment by stffn** [comment]
It's great to see the ongoing community contributions three years after the last commit. decl_auth has been unmaintained since then. I now added this info to the description. I appreciate forks and rewrites for modern day Rails. Since I don't have the resources to review any, I can't recommend any, though.

**Comment by viacommits** [comment]
Hello @stffn!! Thanks for commenting, I hope things are going well for you.  That's understandable that you don't want to officially recommend any.  Instead of endorsing a single replacement, what would you think about merging a PR that includes a notice at the top of the Readme along with a link to a Wiki page listing some popular forks, rewrites, and alternatives?  That way you only have to merge one last PR, and the community can still "maintain" a list of the best options, utilizing this repository's reputation, SEO, etc.  Seems like a win-win to me.

**Comment by niciliketo** [comment]
We are in the process of migrating to Rails 5 and have done the work to ensure DA continues to work for us.
You can find our fork here -
https://github.com/xymist/declarative_authorization
